.ng-dev/release.ts

```diff
@@ -15,6 +16,10 @@ export const release: ReleaseConfig = {
   },
   publishRegistry: 'https://wombat-dressing-room.appspot.com',
   releasePrLabels: ['action: merge'],
-  // TODO: Set up package building.
-  buildPackages: async () => [],
+  buildPackages: () => {
+    // The buildTargetPackages function is loaded at runtime as the loading the script causes an
+    // invocation of bazel.
+    const { buildTargetPackages } = require(join(__dirname, '../scripts/package-builder.js'));
+    return buildTargetPackages('dist/release-output', false, 'Release', /* isRelease */ true);
```

Comment:
I believe there is an extra param here, I think this was copied over from framework where there was an Ivy flag involved as well.

```suggestion
    return buildTargetPackages('dist/release-output', 'Release', /* isRelease */ true);
```